contextManager/index.js

```diff
@@ -476,6 +476,7 @@ class ContextHandler {
 
 let zkHost = process.env.ZOOKEEPER_HOST || "zookeeper";
 let zkPort = process.env.ZOOKEEPER_PORT || 2181;
+let loggerPort = process.env.LOGGER_PORT || 80;
```

Comment:
Hum... I don't know if `LOGGER_PORT` is a good name, actually the webserver has only the logger endpoint, but others endpoints can be added in a near future. Maybe a `SERVICE_PORT` or some other thing more meaningful

Reply:
Hi @raulnegreiros, thank you for your feedback. I changed the environment variable name.